lib/common.ex

```diff
@@ -99,4 +99,14 @@ defmodule ScenicNew.Common do
       Mix.raise("Please select another directory for installation")
     end
   end
+
+  def base_module, do: app_base(otp_app())
+  defp otp_app, do: Mix.Project.config() |> Keyword.fetch!(:app)
+
+  defp app_base(app) do
+    case Application.get_env(app, :namespace, app) do
+      ^app -> app |> to_string() |> Macro.camelize()
+      mod -> mod |> inspect()
+    end
+  end
```

Comment:
I found this documentation:
https://hexdocs.pm/phoenix/Mix.Tasks.Phx.Routes.html
Such configuration does not interest us, so we should remove it.

Also this fixes last warning from `Codecov`:
```suggestion
  def base_module do
    Mix.Project.config() |> Keyword.fetch!(:app) |> Atom.to_string() |> Macro.camelize()
  end
```